1-WebApp-OIDC/1-5-B2C/Views/Shared/_LoginPartial.cshtml

```diff
@@ -21,7 +21,7 @@ else
 {
 <ul class="nav navbar-nav navbar-right">
 	<li class="navbar-btn">
-		<form method="get" asp-area="AzureADB2C" asp-controller="Account" asp-action="SignIn">
+		<form method="get" asp-area="MicrosoftIdentity" asp-controller="Account" asp-action="SignIn">
 			<button type="submit" class="btn btn-primary">Sign In/Up</button>
```

Comment:
>Sign In/Up [](start = 49, length = 10)

sign-up should come first (this is feedback from B2C docs)...so should appear as : Sign Up/In :)